shared/lib/create-redux-store/index.js

```diff
@@ -9,8 +10,11 @@ import { createStore, applyMiddleware } from 'redux';
  */
 import reducers from './reducers';
 
+const loggerMiddleware = createLogger();
+
 export default () => {
 	return applyMiddleware(
-		thunkMiddleware
+		thunkMiddleware,
+		loggerMiddleware
```

Comment:
Yep. This got in for WIP debugging purposes (see the bit about Flux→Redux re-dispatching).